Google-Summer-of-Code-2025.md

```diff
@@ -1119,18 +1120,18 @@ We also recommend taking up at least one checkbox item from each of the followin
 
 **Project Description:**
 
-When learners make a mistake on a concept they have previously demonstrated in an earlier part of a lesson, it often makes sense to redirect them back to an earlier card to try and reinforce earlier concepts that the learner may have not fully understood. However, with the current implementation, learners subsequently need to re-answer all the cards between the earlier state and the state they had reached, which is frustrating.
+When learners make a mistake on a concept they have previously demonstrated in an earlier part of a lesson, it often makes sense to redirect them back (or to a parallel flow) to try and reinforce earlier concepts that the learner may have not fully understood. However, in the current app implementation, learners subsequently need to re-answer all the cards between the earlier state and the state they had reached, which is frustrating.
```

Comment:
Suggest replacing "back (or to a parallel flow)" with "back to an earlier card" -- I think that's the main case that we're trying to fix here.